ibis/backends/exasol/compiler.py

```diff
@@ -187,3 +186,6 @@ def visit_DayOfWeekName(self, op, *, arg):
 
     def visit_ExtractQuarter(self, op, *, arg):
         return self.cast(self.f.to_char(arg, "Q"), op.dtype)
+
+    def visit_TypeOf(self, op, *, arg):
```

Comment:
Simple mappings like this can be added to `SIMPLE_OPS`.

Reply:
Nice, thanks.